source/Interfaces/Menu.hpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file Menu.h
+ * @author
+ *
+ *
+ */
+
+#ifndef CSE_491_MENU_HPP
+#define CSE_491_MENU_HPP
+#include <SFML/Graphics.hpp>
+
+namespace i_2D {
+const int MAX_NUMBER_OF_ITEMS_MENU_ITEMS = 3;
```

Comment:
For a value that is known at compile time, you should make it `constexpr` (we'll talk about this more in a couple of weeks).  The `constexpr` keyword indicates that a value is locked in at compile time.  You should also add `static` so that it doesn't complain about multiple definitions across compilation units.

```
static constexpr int MAX_NUMBER_OF_ITEMS_MENU_ITEMS = 3;
```

(also, make sure to indent  this line properly)